src/govuk/components/notification-banner/template.njk

```diff
@@ -0,0 +1,53 @@
+{%- if params.type == "success" or params.type == "error" %}
+  {% set successOrError = true %}
+{% endif %}
+
+{%- if successOrError %}
+  {% set typeClass = "govuk-notification-banner--" + params.type %}
+{% endif %}
+
+{% if params.role %}
+  {% set role = params.role %}
+{% elif successOrError %}
+  {# If type is success or error, the notification banner is focused on page load. Add `role="alert"` to prioritise the information in the notification banner to users of assistive technologies #}
+  {% set role = "alert" %}
+{% else %}
+  {# Otherwise add `role="region"` to make the notification banner a landmark to help users of assistive technologies to navigate to the banner #}
+  {% set role = "region" %}
+{% endif %}
+
+{% if params.tabindex %}
+  {% set tabindex = params.tabindex %}
+{# TODO:  Link below logic to check whether js focus behaviour is enabled (when decision has been made in  https://github.com/alphagov/govuk-frontend/pull/1977#discussion_r500833933 #}
+{% elif successOrError and not(params.tabindex === false)%}
+  {# If type is success or error, add `tabindex="-1"` to make the notification banner focusable #}
+  {% set tabindex = "-1" %}
+{% endif %}
```

Comment:
I think this can be simplified slightly by checking for the presence of `tabindex` (with _any_ value) in the params object, rather than checking if `params.tabindex` is truthy?

```suggestion
{% if 'tabindex' in params %}
  {% set tabindex = params.tabindex %}
{# TODO:  Link below logic to check whether js focus behaviour is enabled (when decision has been made in  https://github.com/alphagov/govuk-frontend/pull/1977#discussion_r500833933 #}
{% elif successOrError %}
  {# If type is success or error, add `tabindex="-1"` to make the notification banner focusable #}
  {% set tabindex = "-1" %}
{% endif %}
```

Reply:
This should be resolved now @36degrees.